hack/run-test-serial-e2e.sh

```diff
@@ -29,6 +27,67 @@ if [ -n "${E2E_SERIAL_SKIP}" ]; then
 	SKIP="-ginkgo.skip=${E2E_SERIAL_SKIP}"
 fi
 
+JUNIT_DIR=""
+REPORT_DIR="/tmp/artifacts/nrop"
+
+# so few arguments is no bother enough for getopt
+while [[ $# -gt 0 ]]; do
+	case "$1" in
+		--setup)
+			SETUP="true"
+			shift
+			;;
+		--teardown)
+			TEARDOWN="true"
+			shift
+			;;
+		--no-run-tests)
+			RUN_TESTS="false"
+			shift
+			;;
+		--no-color)
+			NO_COLOR="-ginkgo.noColor"
+			shift
+			;;
+		--focus)
+			FOCUS="-ginkgo.focus=$2"
+			shift
+			shift
+			;;
+		--skip)
+			SKIP="-ginkgo.skip=$2"
+			shift
+			shift
+			;;
+		--report)
+			REPORT_DIR="$2"
+			shift
+			shift
+			;;
+		--junit)
+			JUNIT_DIR="$2"
+			shift
+			shift
+			;;
+		*)
+			echo "unrecognized option: $1"
+			echo "use the form --opt val and not the form --opt=val"
+			exit 1
+			;;
+	esac
+done
+
+# mandatory
+if [ -z "$REPORT_DIR" ]; then
```

Comment:
I am not familiar with shell lang, but I see all the vars are used within {}, do we need them to wrap REPORT_DIR?